3DAmsterdam/Assets/Netherlands3D/Scripts/Utilities/Billboard.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Netherlands3D.InputHandler;
+using Netherlands3D.ObjectInteraction;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Netherlands3D
+{
+    
+
+    public class Billboard : Interactable
+    {
+        public string[] Row;
+        public int Index;
+        public Action<int> ClickAction;
+
+        GameObject textmeshGameObject;
+
+        private bool isFiltered;
+
+        Renderer ren;
+        SphereCollider col;
```

Comment:
hi bart; the naming convention discourages using abbrevations for identifier names

Reply:
I used renderer and collider but that overrided the existing properties, I will check this

Reply:
changed it..